addons/l10n_co_pos/static/src/js/pos.js

```diff
@@ -0,0 +1,48 @@
+odoo.define('l10n_co_pos.pos', function (require) {
+"use strict";
+
+var models = require('point_of_sale.models');
+
+var _super_posmodel = models.PosModel.prototype;
+models.PosModel = models.PosModel.extend({
+    is_colombian_country: function () {
+        return this.company.country.code === 'CO';
+    },
+    _save_to_server: function () {
```

Comment:
you should not override,  _save_to_server function, but the 'post_push_order_resolve' in paymentScreen (like for french localisation) and do the rpc that will get the "name" and set it as l10n_co_dian.

Reply:
to avoid extra RPC call I did override _save_to_server, by the way, changes have been done. :+1: